README.md

```diff
@@ -48,13 +48,42 @@ Git Proxy is built with a developer-first mindset. By presenting simple-to-follo
 To install Git Proxy, use the [npm](https://www.npmjs.com/) package manager:
 
 ```bash
-$ npm install @finos/git-proxy
+$ npm install -g @finos/git-proxy
+$ git-proxy
 ```
 
 To install a specific version of Git Proxy, append the version to the end of the `install` command:
 
 ```bash
-$ npm install @finos/git-proxy@1.0.0
+$ npm install -g @finos/git-proxy@1.0.0
```

Comment:
@coopernetes - makes sense to update this to `1.0.1` ?

Reply:
@maoo @coopernetes - `1.1.0` is probably better as these changes constitute a minor improvement, rather than a patch.